client/js/components/actionbar/action-bar.js

```diff
@@ -1,14 +1,33 @@
 var React = require('react');
+var Link = require('react-router-component').Link;
+var classnames = require('classnames');
 var AddEvent = require('./add-event');
+var AddEmployee = require('./add-employee');
+var AllActions = require('./all-actions');
 
-var ActionBar = React.createClass({
+
+var Actionbar = React.createClass({
   render: function () {
+
+    var Action;
+    switch (this.props.hashRoute) {
+      case '#actionbar/add-event': Action = AddEvent; break;
+      case '#actionbar/add-employee': Action = AddEmployee; break;
+      default: Action = AllActions;
```

Comment:
just want to check - when the hash is not correct, it looks like this code will display all actions, making the sidebar and the actionbar have links to the same thing, right? is this is intended effect?